src/pages/yourrides/offer/editOptions.story.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react'
+import { storiesOf } from '@storybook/react'
+
+import { MainContent, Content, BottomContent } from 'layout/content'
+import Section from 'layout/section/baseSection'
+import SubHeader from 'subHeader'
+import TextTitle from 'typography/title'
+import Stepper, { StepperDisplay } from 'stepper'
+import Divider from 'divider'
+import ItemCheckbox from 'itemCheckbox'
+import ItemInfo from 'itemInfo'
+import Textarea from 'textarea'
+import ComfortIcon from 'icon/comfortIcon'
+import LadyIcon from 'icon/ladyIcon'
+import LightningIcon from 'icon/lightningIcon'
+import Button, { ButtonStatus } from 'button'
+
+const stories = storiesOf('Pages|Your rides/Offer/Edit/Options', module)
+
+type EditOptionsState = Readonly<{
+  isComfort: boolean
+  isLadyOnly: boolean
+  isInstantBooking: boolean
+  buttonStatus: ButtonStatus
+}>
+
+class EditOptions extends Component<{}, EditOptionsState> {
+  state: EditOptionsState = {
+    isComfort: false,
+    isLadyOnly: false,
+    isInstantBooking: false,
+    buttonStatus: ButtonStatus.PRIMARY,
+  }
+
+  save = (): void => {
+    this.setState({ buttonStatus: ButtonStatus.LOADING })
+
+    setTimeout(() => this.setState({ buttonStatus: ButtonStatus.CHECKED }), 1000)
+    setTimeout(() => this.setState({ buttonStatus: ButtonStatus.PRIMARY }), 3000)
+  }
+
+  updateOption = (name: string): void => {
+    this.setState((prevState) => ({ [name]: !prevState[name] }))
+  }
+
+  render = (): JSX.Element => (
+    <MainContent>
+      <Content>
+        <Section>
+          <SubHeader>Passengers options</SubHeader>
+
+          <div style={{ display: 'flex' }}>
+            <TextTitle style={{ flex: 1, lineHeight: '48px' }}>Number of passengers</TextTitle>
+            <div style={{ position: 'relative', right: '-14px' }}>
```

Comment:
`right: '-14px'` is needed for the + icon of the `Stepper` to be aligned with the `ItemCheckbox` circles... I don't know how to improve this :(

Reply:
As discussed, I would either create a new ItemStepper widget or add new props to the exisiting stepper.
As long as, we don't do CSS in kairos, I'm ok with both approaches.

Reply:
As discussed, I would either create a new ItemStepper widget which will be an Item that contains a Stepper. The Item knows how to manage its horizontal padding automatically.